src/js/common/components/questions/FullName.jsx

```diff
@@ -111,6 +115,8 @@ FullName.propTypes = {
       dirty: React.PropTypes.bool,
     }),
   }).isRequired,
+  customValidation: React.PropTypes.func,
```

Comment:
I was going to say we should leave off `custom` in these two properties, but it's probably better to leave it, since they work differently than when we pass `errorMessage` and `validation` to other components.

I think it would make sense to move towards validation functions that return the message and the status. The way we put ternary statements in these `errorMessage` props doesn't work well for components with a couple different validation errors. Not something for this PR, but should think about it in the future.